### ISSUE TITLE
Histograms ignore `ylab` argument

### ISSUE THREAD

**Issue by lbelzile** [grantmcdermott/tinyplot#203]
The histogram type in `tinyplot` ignores the `ylab` argument, unlike `hist`. Is this behaviour intentional?

MWE:

``` r
y <- rexp(100)
par(mfrow = c(1, 2))
tinyplot::plt(x = y, 
              type = "hist", 
              xlab = "mylabel", 
              ylab = "text")
hist(x = y, 
     xlab = "mylabel", 
     ylab = "text")
```

![](https://i.imgur.com/NoQElzC.png)

<sup>Created on 2024-08-14 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

**Comment by vincentarelbundock** [comment]
@lbelzile thanks for the report. I created a Pull Request to fix this. You can try it with:

```r
remotes::install_github("vincentarelbundock/tinyplot@issu203")
```

Make sure you restart R before trying again.

@grantmcdermott this is a really trivial fix. Do you want to review, or should I just merge? (I'm happy to wait, just wondering what your preferred code review policy is.)

**Comment by grantmcdermott** [comment]
Thanks @vincentarelbundock. Happy for you to merge if you are satisfied and all checks pass. 

(Thanks too for the report @lbelzile.)

**Comment by grantmcdermott** [comment]
Nvm. Had some time to check and merge myself. Closed by #204.
